morpheus/_lib/include/morpheus/utilities/type_util.hpp

```diff
@@ -60,4 +60,5 @@ DType type_to_dtype()
     return DType::from_triton(cudf::type_to_id<T>);
 }
 
+/** @} */  // end of group
```

Comment:
Is this missing a start of group tag?

Reply:
good catch. It should be under `utilities` group. thanks